spec/views/users/index_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+RSpec.describe 'Index page', type: :feature do
+  before(:each) do
+    @user = User.create(
+      name: 'John',
+      bio: 'Software Developer',
+      photo: 'http://hello1.com/org.png',
+      posts_counter: 0
+    )
+    @user_two = User.create(
+      name: 'Victor',
+      bio: 'Graphic designer',
+      photo: 'http://hello1.com/org.png',
+      posts_counter: 0
+    )
+    visit '/users'
+  end
+  describe 'index page' do
+    it 'shows the right content' do
+      expect(page).to have_content('John')
+      expect(page).to have_content('Victor')
+    end
```

Comment:
- [ ] Please add tests that asserts that you can see the number of posts each user has written.
![image](https://user-images.githubusercontent.com/46839548/223721324-9402ae3e-5b99-4409-94e2-24866b4ed0c3.png)